.travis.yml

```diff
@@ -0,0 +1,33 @@
+language: android
+
+android:
+  components:
+    - platform-tools
+    - build-tools-23.0.1
+    - android-23
+    - extra-android-m2repository
+    - sys-img-armeabi-v7a-android-19
+
+before_cache:
+    - rm -f $HOME/.gradle/caches/modules-2/modules-2.lock
+
+cache:
+    directories:
+        - node_modules
+        - .nvm
+        - $HOME/.gradle/caches/
+        - $HOME/.gradle/wrapper/
+
+install:
+    - nvm install 4
+    - npm install -g nativescript --ignore-scripts
+    - tns usage-reporting disable
+    - tns platform add android
+
+before_script:
+    - echo no | android create avd --force -n test -t android-19 -b armeabi-v7a
+    - emulator -avd test -no-audio -no-window &
+    - android-wait-for-emulator
+
+script:
+    - tns test android --emulator
```

Comment:
If the emulator is already started, you do not need `--emulator` option (for Android)